ls1intum/Artemis
# `Exam mode`: Show number of accepted complaints in assessment dashboard

### Checklist
#### General


#### Client

### Motivation and Context


Fixes #6419 
in the assessment dashboard for exam exercises, the number of accepted complaints is not displayed. The reason for this problem is that the number of accepted complaints is only displayed if the property `course` is present, but exam exercises don't have this property.

### Description

- For exam exercises we need to use the course of the exam, not the course of the exercise.
- In exam mode, the property `isAtLeastInstructor` is not a property of the course but of the exercise, so we need to check for that too
- The URL for exam complaints is different from the one for normal exercises, so that needs to be adapted.

### Steps for Testing

1. Create an exam with at least 1 exercise
2. As a student: Participate in the exam
3. As a tutor: Evaluate the submission
4. As student: File a complaint
5. As a tutor: Accept the complaint
6. As an instructor: Make sure that the tutor leaderboard is displayed correctly in the exam dashboard and the exercise assessment dashboard. Verify that the number of accepted complaints links to the correct page.
7. As a tutor: Verify that the number of accepted complaints does not link to the complaint list.

### Review Progress






#### Code Review
#### Manual Tests
